src/System.Collections/src/System/Collections/BitArray.cs

```diff
@@ -74,40 +77,40 @@ public BitArray(byte[] bytes)
             // this value is chosen to prevent overflow when computing m_length.
             // m_length is of type int32 and is exposed as a property, so 
             // type of m_length can't be changed to accommodate.
-            if (bytes.Length > int.MaxValue / BitsPerByte)
+            if (bytes.Length > int.MaxValue >> BitShiftPerByte) // Divide by 8.
```

Comment:
Does this actually help?  Isn't it just computed as a constant at compile time?

Reply:
Yes, good point, will revert anywhere we have constants.